README.md

```diff
@@ -89,6 +89,19 @@ plot = PlotAI(df)
 plot.make("make a scatter plot")
 ```
 
+By default the library will use '*gpt-3.5-turbo*'. You can use different OpenAI models:
+
+```python
+# import PlotAI
+from plotai import PlotAI
+
+# create PlotAI object, pass pandas DataFrame as an argument
+plot = PlotAI(df)
```

Comment:
I would rather set model in `PlotAI` constructor, what do you think?

Reply:
Yes, it probably makes more sense.